pkg/utils/utils.go

```diff
@@ -121,6 +124,20 @@ func UmountDisk(path string) error {
 	return os.NewSyscallError("umount", err)
 }
 
+func ForceUmountWithTimeout(path string, timeout time.Duration) error {
+	isHostProcMounted, err := IsHostProcMounted()
+	if err != nil {
+		return err
+	}
+	if isHostProcMounted {
+		_, err := executeOnHostNamespaceWithTimeout("umount", []string{path, "-f"}, timeout)
```

Comment:
I think it is best practice to set the command options first and at last the device file.
```suggestion
		_, err := executeOnHostNamespaceWithTimeout("umount", []string{"-f", path}, timeout)
```

Reply:
Nice catch, updated!